plume-models/src/blogs.rs

```diff
@@ -236,6 +243,10 @@ impl Blog {
                     .public_key_publickey()?
                     .public_key_pem_string()?,
                 private_key: None,
+                // TODO: federate the two following properties
```

Comment:
or all three?

Reply:
Yes, at the time I wrote the comment `summary_html` was not here yet. :grin: